CONTRIBUTING.rst

```diff
@@ -59,51 +59,48 @@ Get Started!
 
 Ready to contribute? Here's how to set up `weather_forecast_retrieval` for local development.
 
-1. Fork the `weather_forecast_retrieval` repo on GitHub.
-2. Clone your fork locally::
+#. Fork the `weather_forecast_retrieval` repo on GitHub to your user and check
+   out the repository locally.
 
-    $ git clone git@github.com:your_name_here/weather_forecast_retrieval.git
-
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+#. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper
+   installed, this is how you set up your fork for local development::
 
     $ mkvirtualenv weather_forecast_retrieval
     $ cd weather_forecast_retrieval/
     $ python setup.py develop
 
-4. Create a branch for local development::
+#. Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
    Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox::
+#. When you're done making changes, check that your changes pass flake8 and
+   all the tests::
 
     $ flake8 weather_forecast_retrieval tests
-    $ python setup.py test or py.test
-    $ tox
-
-   To get flake8 and tox, just pip install them into your virtualenv.
+    $ python -m unittest discover
 
-6. Commit your changes and push your branch to GitHub::
+#. Commit your changes and push your branch to GitHub::
 
     $ git add .
     $ git commit -m "Your detailed description of your changes."
     $ git push origin name-of-your-bugfix-or-feature
 
-7. Submit a pull request through the GitHub website.
+#. Submit a pull request through the GitHub website.
 
 Pull Request Guidelines
 -----------------------
 
 Before you submit a pull request, check that it meets these guidelines:
 
-1. The pull request should include tests.
-2. If the pull request adds functionality, the docs should be updated. Put
+#. The pull request should include tests.
+#. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.6, 2.7, 3.3, 3.4 and 3.5, and for PyPy. Check
-   https://travis-ci.org/scotthavens/weather_forecast_retrieval/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+#. Once opened, every pull request will be tested against every supported
+   Python version through GitHub actions. Check the tab if there are any issues
+   and that all workflows pass.
 
 Tips
```

Comment:
Could you add this:

Releasing to Pypi
-------------------

To create a new release on Pypi.org, follow these steps:

1. Create a new release for `weather_forecast_retrieval`
2. Name the tag and release the version number, for example `v0.7.0`
3. Add documentation about the release and why it's different from the previous
4. Publish new release which will trigger a build to release to Pypi